README.md

```diff
@@ -118,20 +118,14 @@ registry:yourrepo/yourimage:tag        pull image directly from a registry (no c
 
 #### Basic Grype Vulnerability Data Shape
 ```json
- {
-  "vulnerability": {
-    ...
-  },
-  "relatedVulnerabilities": [
-    ...
-  ],
-  "matchDetails": [
-    ...
-  ],
-  "artifact": {
-    ...
-  }
-}
+"matches": [
+    {
+        "vulnerability"         : { ... },
+        "relatedVulnerabilities": [ ... ],
+        "matchDetails"          : [ ... ],
+        "artifact"              : { ... }
+    },  ...
+], 
```

Comment:
I agree that this section needs some cleanup. 👍  I'd say this entire section should be moved down to the "output" section, since it applies to only our JSON output.

I see that you're using this as a reference for the template data structure. This isn't a comprehensive description of that data structure, so I think it's a good idea not to point people here for working on templates.

I'd suggest leaving this change out of this PR for now.